core/src/main/java/org/lflang/LinguaFranca.xtext

```diff
@@ -200,10 +200,11 @@ Action:
 Reaction:
     (attributes+=Attribute)*
     (('reaction') | mutation ?= 'mutation')
+    (name=ID)?
     ('(' (triggers+=TriggerRef (',' triggers+=TriggerRef)*)? ')')
-    (sources+=VarRef (',' sources+=VarRef)*)?
+    ( => sources+=VarRef (',' sources+=VarRef)*)?
```

Comment:
This is to tell ANTLR to always prefer going down this path rather than use the `Connection` production.